app/routes/_mainLayout.t7_.$character.$move.tsx

```diff
@@ -40,38 +42,48 @@ export const loader = async ({ params }: DataFunctionArgs) => {
     });
 
     if (!response.data.values) {
```

Comment:
Good catch that this error will never be returned the the user. But would it be even better to move this check out of the try catch block?
Could move the check ot line 50 in the original code, then we dont have to rethrow in the catch?

Reply:
By moving the `if (!response.data.values)` to line 50, then we need to get rid of the `try/catch`, since we do not have access to the `response` variable outside the `try` block. 

Then we need to do something like this instead:

```typescript 
    const response = await sheets.spreadsheets.values.get({
      spreadsheetId: "1p-QCqB_Tb1GNX0KaicHr0tZwKa1taK5XeNvMr1N3D64",
      range: character, // sheet name
    }).catch(() => {
      throw new Response(null, { status: ServerStatusCode.ServerError, statusText: "server error" });
    });
    
    if (!response.data.values) {
      throw json("not found", { status: ServerStatusCode.NotFound, statusText: "Not found 1" });
    }
```

Reply:
Yes, I think that would be a good solution :)